README.md

```diff
@@ -1 +1,194 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 LEADERBOARD <a name="about-project"></a>
+
+**Leaderboard** is a project I built to display results scored by players in the game competition .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Javascript  <a name="Javascript"></a>
+### API <a name="API"></a>
+### Gitflow <a name="Gitflow"></a>
+### VS code <a name="VS code"></a>
+### Webpack <a name="Webpack"></a>
+### HTML/CSS <a name="HTML/CSS"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/ikoote1/Leaderboard/edit/main/README.md">Readme</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/ikoote1/Leaderboard">Repo</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Result display**
+- **Form for the result input**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="Preim Live"></a>
+
+- Live Demo Link(href="#")
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+-**You are allowed to clone this project**
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+>To Clone this (https://github.com/ikoote1/Leaderboard.git) repository to your desired folder:
+
+### Setup
+
+>Clone this (https://github.com/ikoote1/Leaderboard.git) repository to your desired folder:
+
+
+  git clone git@github.com/ikoote1/Leaderboard.git
+
+
+### Install
+
+Install this project with:
+
+-Link (href="https://ikoote1.github.io/Leaderboard/)
+
+### Usage
+  
+> Clone the project from github
+> Or use the link to live demo and access it from there.
+
+### Run tests
+
+To run tests, run the following steps:
+
+> Check the live demo and check on how the project work.
+> Clone and test for linter errors.
+> Check the dynamics.
+
+### Deployment
+
+-  This project was deployed using github pages and this the <a href="https://ikoote1.github.io/Leaderboard/">link</a> to the live devmo.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Ikoote Rasuli"></a>
+
+> IKOOTE RASULI
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ikoote1)
+- Twitter: [@twitterhandle](https://twitter.com/ikoote1)
+- LinkedIn:[@LinkedIn](https://www.linkedin.com/public-profile/settings?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_self_edit_contact-info%3B4xL98PtKS7SDj%2Fe93TEh9w%3D%3D)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] Kindly consider removing this line as it is a readme template guideline and refers to the author of the repo and not to the visitor of your readme. removing it will make your repo professional.